recipe/meta.yaml

```diff
@@ -21,42 +28,51 @@ requirements:
     - make  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
+    # pyarrow does not require nvcc but it needs to link against libraries in arrow-cpp=*=*cuda
+    - {{ compiler("cuda") }}  # [cuda_compiler_version not in (undefined, "None")]
   host:
     # directly pin boost-cpp as we also seem to directly include boost symbols
     # in the Python modules.
-    - arrow-cpp {{ version }}
+    - arrow-cpp ={{ version }}=*{{ build_ext }}
+    # The version of re2 should match with the version that arrow-cpp
+    # was built against:
+    - re2 2020.04.*
     - boost-cpp
     - cython
     - numpy 1.16.*
     - python
     - setuptools
     - setuptools_scm
-    - six
-
   run:
     - {{ pin_compatible('numpy', lower_bound='1.16') }}
     - boost-cpp
-    - futures  # [py==27]
+    - arrow-cpp ={{ version }}=*{{ build_ext }}
+    - re2 2020.04.*
```

Comment:
With https://github.com/conda-forge/cf-mark-broken/pull/53 we should be able to remove `re2` dependency.

Reply:
```suggestion
```